### DISCUSSION TITLE
rockchip系列芯片设备如何备份还原系统

### DISCUSSION THREAD

**Discussion by maxmib** [ophub/amlogic-s9xxx-armbian#2014] [Rockchip]
1.armbian-ddbr备份的镜像是否能用于直接写入emmc还原rockchip芯片设备
2.armbian-ddbr备份的镜像是否能直接用rkdevtool线刷进设备
3.对于未刷过机的rockchip设备，armbian-install似乎无法使用，是否能用armbian-ddbr写入同型号设备刷机后的备份镜像达到刷机目的
谢谢

**Comment by 13584452567** [comment]
1.可以
2.可以
3.可以
但是不推荐，ddbr备份出来的文件太大，不管是dd还是rk的工具线刷，都等于全盘写入一次。如果想批量写入设备，可以使用rk的工厂烧录工具。
